src/selectors/entities/channels.ts

```diff
@@ -892,3 +892,11 @@ export function isManuallyUnread(state: GlobalState, channelId?: string): boolea
 
     return Boolean(state.entities.channels.manuallyUnread[channelId]);
 }
+
+export function getChannelModerations(state: GlobalState, channelId: string): Array<ChannelModeration> | object {
```

Comment:
```suggestion
export function getChannelModerations(state: GlobalState, channelId: string): Array<ChannelModeration> | Array<[]> {
```

Reply:
Actually just dropping the object and responding with an array works since `[]` fulfills `Array<ChannelModeration>`

Reply:
will also add a test for this state